webapp/channels/src/components/admin_console/team_channel_settings/group/index.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {bindActionCreators} from 'redux';
+
+import type {Dispatch} from 'redux';
+
+import {t} from 'utils/i18n';
+
+import type {GlobalState} from 'types/store';
+
+import GroupList from './group_list';
+import type {Props} from './group_list';
+
+import type {GenericAction} from 'mattermost-redux/types/actions';
+
+import type {Group} from '@mattermost/types/groups';
+
+function mapStateToProps(state: GlobalState, ownProps: Props) {
```

Comment:
The Own Props can be defined here in this file and then can be exported to GroupList to form the final Props

```js
interface Props extends OwnProps {
..
..
}
```